### ISSUE TITLE
[BUG] Single mode trivia always looses

### ISSUE THREAD

**Issue by LeonTheDev-io** [aniket091/Gamecord#14]
**Describe the bug**
Single mode trivia games always tell you, you lost, even if your answer is correct

**To Reproduce**
1. Create a new single mode trivia game
2. Get a correct answer

**Expected behavior**
It should tell you 'you won' if you won

**Package versions:**
- Node Version: [18.14.0]
- Gamecord Version: [4.2.0]
- Discord.js Version: [14.6.0]

**Comment by retrouser955** [comment]
Hello @LeonTheDev-io! Could you provide the code you used for trivia?

**Comment by LeonTheDev-io** [comment]
```js
if (game == 'trivia') {
        const Game = new Trivia({
          message: interaction,
          isSlashGame: true,
          embed: {
            title: 'Trivia',
            color: '#5865F2',
            description: 'You have 60 seconds to guess the answer.',
          },
          timeoutTime: 60000,
          buttonStyle: 'PRIMARY',
          trueButtonStyle: 'SUCCESS',
          falseButtonStyle: 'DANGER',
          mode: interaction.options.getString('mode'),
          difficulty: interaction.options.getString('difficulty'),
          winMessage: 'You won! The correct answer is {answer}.',
          loseMessage: 'You lost! The correct answer is {answer}.',
          errMessage: 'Unable to fetch question data! Please try again.',
          playerOnlyMessage: 'Only {player} can use these buttons.',
        });
        Game.startGame();
      }
```

**Comment by aniket091** [comment]
Fixed this issue in `v3.3.0 & v4.3.0`, update to the lastest version. 🙂
Thanks for reporting the bug. 🐛
